.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,27 @@
-# What's this PR do?
+# [NaN-1](https://github.com/nanlabs/python-reference/issues/1)
 
-_Summary of changes in this PR or what it accomplishes._
+## Story
 
-@ulises-jeremias
+Here we describe what the product owner wants.
+
+## Acceptance criteria
```

Comment:
Umm, I don't think that an Acceptance Criteria is something we associate with a PR. Maybe we can use this template for issues template (feature requests). I think we can revert the changes to this file for now to let DangerJS pass and then discuss a new template on another PR 👍🏻